MAUI/Autocomplete/Selection.md

```diff
@@ -394,4 +394,41 @@ In the Autocomplete control, the drop-down can be opened or closed programmatica
 autocomplete.IsDropDownOpen = true;
 
 {% endhighlight %}
+{% endtabs %}
+
+
+## Clear() method support
+
+Clear the selected values in the Autocomplete using the Clear() method as shown in the code snippet below:
+
+{% tabs %}
```

Comment:
Just add the code used to clear items. No need of elongated codes.

Reply:
Updated